src/app/tests/suites/certification/Test_TC_S_2_4.yaml

```diff
@@ -0,0 +1,359 @@
+          supporting scenes."
+      verification: |
+          Is DUT configured with AC2?
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value:
+                    "Please configure AC2 on DUT and enter 'y' after the
+                    configuration is complete"
+              - name: "expectedValue"
+                value: "y"
+
+    - label:
+          "TH sends a RecallScene command to DUT with the GroupID field set to
+          0x0001 and the SceneID field set to 0x01 and the TransitionTime field
+          omitted."
+      PICS: S.S.C05.Rsp
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 0x0032
+
+    - label: "DUT transitions to AC1 over 5."
+      verification: |
+          Is the transition done over 5s?
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value: "Please confirm that transition to AC1 was made over 5s"
+              - name: "expectedValue"
+                value: "y"
+
+    - label:
+          "TH configures AC2 on DUT for all implemented application clusters
+          supporting scenes."
+      PICS: PICS_SDK_CI_ONLY
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 200
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label:
+          "TH configures AC2 on DUT for all implemented application clusters
+          supporting scenes."
+      verification: |
+          Is DUT configured with AC2?
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value:
+                    "Please configure AC2 on DUT and enter 'y' after the
+                    configuration is complete"
+              - name: "expectedValue"
+                value: "y"
+
+    - label:
+          "TH sends a RecallScene command to DUT with the GroupID field set to
+          0x0001 and the SceneID field set to 0x01 and the TransitionTime field
+          omitted."
+      PICS: S.S.C05.Rsp
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: null
+      response:
+          error: 0x00
```

Comment:
Something needs to check that the transition to AC1 happened: automated step in CI, manual otherwise.